tests/ballerina-test-utils/src/main/java/org/ballerinalang/test/BCompileUtil.java

```diff
@@ -58,7 +59,9 @@ public static CompileResult compile(String sourceFilePath) {
         Package currentPackage = project.currentPackage();
         PackageCompilation packageCompilation = currentPackage.getCompilation();
         JBallerinaBackend jBallerinaBackend = JBallerinaBackend.from(packageCompilation, JdkVersion.JAVA_11);
-        if (jBallerinaBackend.hasDiagnostics()) {
+        long errorCount = jBallerinaBackend.diagnostics().stream().filter(diagnostic ->
```

Comment:
We can use `findAny` function instead finding the count since there is no usage of the count - https://www.baeldung.com/java-stream-findfirst-vs-findany

Reply:
@hemikak WDYT of using `anyMatch(predicate)` instead? IMO it'll be more convenient for the above context, if we don't need the exact count.

Reply:
+1